shared_model/backend/protobuf/query_responses/impl/proto_account_response.cpp

```diff
@@ -36,11 +36,11 @@ namespace shared_model {
         : AccountResponse(std::move(o.proto_)) {}
 
     const interface::Account &AccountResponse::account() const {
-      return *account_;
+      return account_;
     }
 
     const AccountResponse::AccountRolesIdType &AccountResponse::roles() const {
-      return *accountRoles_;
+      return accountRoles_;
```

Comment:
Please fix the naming of all the fields like that. Please use snake_case_ for them